server/src/main.go

```diff
@@ -1426,6 +1439,38 @@ func (c MainController) SupporterSpokeCSVHandler(w http.ResponseWriter, r *http.
 	writer.Flush()
 }
 
+func (c MainController) NewActivistSpokeCSVHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
could you make "Activist" plural (NewActivist**s**SpokeCSVHandler) it does change the meaning a bit!